code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1270,6 +1270,21 @@
 	. = 1
 	..()
 
+/datum/reagent/medicine/regen_jelly/regen_ooze
+	name = "Regenerative Ooze"
+	description = "Gradually regenerates all types of damage, without harming slime anatomy."
+	reagent_state = LIQUID
+	color = "#65d891"
+	taste_description = "jelly"
+
+/datum/reagent/medicine/regen_jelly/regen_ooze/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-0.5*REM, 0)
+	M.adjustFireLoss(-0.5*REM, 0)
+	M.adjustOxyLoss(-0.5*REM, 0)
+	M.adjustToxLoss(-0.5*REM, 0, TRUE) //heals TOXINLOVERs
+	. = 1
+	..()
```

Comment:
The biggest problem with this is: `/datum/reagent/medicine/regen_jelly/regen_ooze/on_mob_life` calls its parent proc, `/datum/reagent/medicine/regen_jelly/on_mob_life`, and the parent proc heals *again*; i.e. it heals 2 every tick. Just make it `/datum/reagent/medicine/regen_ooze`, or make a var denoting the healing power so it can OOPed properly.